### PR TITLE
Rename Alias to Account

### PR DESCRIPTION
**Pull request by thibault-martinez**
Fixes https://github.com/iotaledger/iota-sdk/issues/1037

### CODE REVIEW

File: bindings/nodejs/lib/types/wallet/account.ts [hunk 1]
```diff
@@ -91,11 +91,11 @@ export interface SyncOptions {
...
     /** Specifies what outputs should be synced for the ed25519 addresses from the account. */
     account?: AccountSyncOptions;
-    /** Specifies what outputs should be synced for the address of an alias output. */
-    alias?: AliasSyncOptions;
+    /** Specifies what outputs should be synced for the address of an account output. */
+    account?: AccountSyncOptions;
     /** Specifies what outputs should be synced for the address of an nft output. */
     nft?: NftSyncOptions;
-    /** Specifies if only basic outputs with an AddressUnlockCondition alone should be synced, will overwrite `account`, `alias` and `nft` options. Default: false. */
+    /** Specifies if only basic outputs with an AddressUnlockCondition alone should be synced, will overwrite `account`, `account` and `nft` options. Default: false. */
```
Comment (DaughterOfMars) [inline bindings/nodejs/lib/types/wallet/account.ts]: hmmmmmm how did this happen

Comment (thibault-martinez) [reply bindings/nodejs/lib/types/wallet/account.ts]: I think it's technically correct but we just have this account/account name clash, will have a look

Comment (thibault-martinez) [reply bindings/nodejs/lib/types/wallet/account.ts]: Reverted

**Review by DaughterOfMars — APPROVED** [review]

File: bindings/nodejs/examples/how_tos/account_wallet/transaction.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ async function run() {
         }
 
         const syncOptions = {
-            alias: {
+            account: {
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/account_wallet/transaction.ts:27]: In the type definition and Rust files you undid this rename

**Review by DaughterOfMars — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

File: bindings/nodejs/examples/how_tos/account_wallet/transaction.ts (hunk 2 above)
Comment (Thoralf-M) [reply bindings/nodejs/examples/how_tos/account_wallet/transaction.ts:27]: Can be done later :trollface:

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by abdulmth — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**